libraries/FusionData/FusionData.h

```diff
@@ -1,12 +1,13 @@
+#ifndef FUSIONDATA
```

Comment:
By Tradition, header exclusion constants get _H on the end (i.e. FUSIONDATA_H here) so they don't accidentally collide with normal constants.